apps/frontend/src/locales/en/translations.json

```diff
@@ -429,6 +429,12 @@
             "description": "The decentralized bitcoin trading and lending platform",
             "cta": "<0>New to Sovryn?</0>",
             "ctaText": "New to Sovryn?"
+        },
+        "protocolDataSection": {
+            "title": "Protocol Data",
+            "cta": "View more",
+            "totalValueLocked": "Total value locked",
+            "volume": "24 hour volume"
```

Comment:
These labels are capitalized in PRD, (you used the correct option in the title)

Reply:
You're right, it's probably incorrect in the design because we have them capitalized in the alpha dapp as well.